src/main/java/mezz/jei/plugins/debug/JeiDebugPlugin.java

```diff
@@ -70,7 +68,7 @@ public void registerCategories(IRecipeCategoryRegistration registration) {
 	}
 
 	@Override
-	public void registerRecipes(IRecipeRegistration registration) {
+	public void registerRecipes(mezz.jei.api.registration.IRecipeRegistration registration) {
```

Comment:
Lines like this shouldn't be changed, imports go at the top

Reply:
Heh, my Intellij has been having some trouble with imports lately, sorry for these weird import issues.